shared/css/color.scss

```diff
@@ -229,6 +229,11 @@ $light_aqua_100: #CEFFFD;
 $light_aqua_500: #3CFFF8;
 $light_aqua_700: #30CCC6;
 
```

Comment:
Why do we need these here?
The idea of dsco variables is to use them as much as possible and shared/colors as less as possible